src/arweave.rs

```diff
@@ -777,16 +778,21 @@ impl Arweave {
     pub async fn download_transaction_data<Context, HttpClient, Output>(
         &self,
         ctx: &Context,
+        concurrency_level: u16,
         tx: &TransactionId,
         output: &mut Output,
         peer: Option<Url>,
+        retries_per_chunk: Option<u16>,
+        // concurrency_level: Option<u16>
     ) -> Result<(), ArweaveError>
     where
         Context: ClientAccess<HttpClient>,
         HttpClient: Client<Request = reqwest::Request, Response = reqwest::Response>,
         HttpClient::Error: From<reqwest::Error>,
         Output: AsyncWrite + AsyncSeek + Unpin,
     {
+        let retries_per_chunk = if retries_per_chunk.is_some() { retries_per_chunk.unwrap() } else { DEFAULT_RETRIES_PER_CHUNK };
```

Comment:
You can use `retries_per_chunk.unwrap_or(DEFAULT_RETRIES_PER_CHUNK)` to simplify the code.